README.md

```diff
@@ -1,5 +1,92 @@
 # Serverless Workflow Specification - Typescript SDK
 
-Provides (TODO: add specifics) for the [Serverless Workflow Specification](https://github.com/serverlessworkflow/specification)
+Provides the Java API/SPI for the [Serverless Workflow Specification](https://github.com/serverlessworkflow/specification)
+
+
+With the SDK you can:
+* Parse workflow JSON and YAML definitions
+* (_WIP_) Programmatically build workflow definitions
+
+
+## Getting Started
+
+
+### Building locally
+
+To build the project and run tests locally:
+
+```
+git clone https://github.com/serverlessworkflow/sdk-typescript.git
+npm install && npm run test
+```
+
+
+### Add as dependency to your project
+```sh
+npm install sdk-typescript
```

Comment:
I am assuming here that we are going to publish the package into npm (`sdk-typescript` or `@serverlessworkflow/sdk-typescript`

Reply:
agreed, but let's leave this out until we deploy it there
+1 for @serverlessworkflow/sdk-typescript

Reply:
makes sense, 
is it ok if I add a description on how to do it with [npm-link](https://docs.npmjs.com/cli/v7/commands/npm-link)?

Reply:
sure! thanks